reference/commands/misc/inspect.rst

```diff
@@ -68,3 +68,7 @@ If no specific attributes are defined via ``-a``, then, some default attributes
     apply_env: True
     build_policy: None
     topics: None
+    settings: ('os', 'arch', 'compiler', 'build_type')
+    options
```

Comment:
Why it is not `options:` ?

Reply:
I need to check inspect output, but your argument is authentic.

Reply:
Done!